### PR TITLE
created HTML submit form for business app

### PR DESCRIPTION
**Pull request by MatinMirzaei**

### CODE REVIEW

**Review by edmundchan8 — APPROVED** [review]
I approve

**Review by SkutchWill — DISMISSED** [review]
Looks like everything we need is there.  Thank you for your hard work!

File: OrcaStarsWebApplication/Views/Home/bussiness_form.cshtml [hunk 1]
```diff
@@ -25,16 +25,16 @@
...
         <br />
         <label>Description:</label>
-        <textarea placeholder="Bussiness Description"></textarea>
+        <textarea placeholder="Business Description"></textarea>
         <br />
         <label>Website:</label>
-        <input type="text" name="BussinessWebsite" placeholder="Bussiness Website">
+        <input type="text" name="BusinessWebsite" placeholder="Business Website">
         <br />
         <label>Cordinates:</label>
```
Comment (edmundchan8) [inline OrcaStarsWebApplication/Views/Home/bussiness_form.cshtml]: Rename:
```
<label>Cordinates:</label>
        <input type="text" name="Cordinates" placeholder="Cordinates">	        <input type="text" name="Cordinates" placeholder="Cordinates">
```
to
```
<label>Coordinates:</label>
        <input type="text" name="Coordinates" placeholder="Coordinates">	        <input type="text" name="Coordinates" placeholder="Coordinates">
```

**Review by edmundchan8 — CHANGES_REQUESTED** [review]
Actually, just typo changed for Coordinates

**Review by edmundchan8 — APPROVED** [review]
Approve the changes!

**Review by madelynmlocke — APPROVED** [review]

**Review by nateptay — CHANGES_REQUESTED** [review]
Still seeing a 404 error when I ran it. It looks like the develop branch you pulled didn't have the fixes to the path yet. The current develop branch has an "Admin" view folder instead of the "Home" view folder. 
Re-naming/removing the index.cshtml page will require some changes to the Startup.cs and AdminController.cs to get to the page. 
If you could change the name of the Home view folder back to Admin that would be good, and I can make the path go to the correct page from the Controller so it can find the form page.

**Comment by Bansenauer-Cascadia** [comment]
This form provides a good baseline.  It is missing an action and would be improved by including a backing ViewModel and the use of Asp tag helpers.

**Pull request closed without merging**